docs/index.html

```diff
@@ -21,14 +21,14 @@ <h2 class="bd-featurette-title">Easy to get started.</h2>
     <p class="lead">Quickly include Bootstrap's CSS and JS into any project. Use your favorite package manager, download the source, or use the Bootstrap CDN.</p>
 
     <div class="row">
-      <div class="col-sm-6 m-b-3">
+      <div class="col-sm-6">
         <h4>Managed dependencies</h4>
         <p>Include Bootstrap's source Sass and JavaScript files via Bower, Composer, Meteor, or npm. Package managed installs don't include documentation, but do include our Gruntfile and readme.</p>
         <p>
           <a class="btn btn-bs btn-outline" href="{{ site.baseurl }}/getting-started/download/#package-managers">View install methods</a>
         </p>
       </div>
-      <div class="col-sm-6 m-b-3">
```

Comment:
Once I removed the `Customized builds` section, these resulted in too much padding.